package.json

```diff
@@ -15,7 +15,8 @@
     "sass:build": "npm run sass:lint && npm run sass:compile && npm run sass:postcss",
     "js:lint": "eslint js/*.js",
     "svg:sprite": "svg-sprite --svg-xmldecl=false --svg-doctype=false --svg-rootattrs=_config/svg-sprite.config.json --log=info --symbol --symbol-prefix='%s' --symbol-example=true --shape-id-generator='cd-icon--' --symbol-dest=img/icons --symbol-sprite=cd-icons-sprite.svg img/icons/*.svg",
-    "e2e": "jest --maxWorkers=2 _tests/*.test.js",
+    "e2e": "jest _tests/*.test.js",
+    "ci": "jest --maxWorkers=2 _tests/*.test.js",
```

Comment:
I think this line is the only change that affects https://github.com/UN-OCHA/common-design-site/pull/90